docs/docsite/rst/contributing/index.rst

```diff
@@ -8,7 +8,7 @@ Contributing Content
    :maxdepth: 2
 
    creating_role
-   creating_multi
+   creating_collections
    creating_apb
    version
    importing
```

Comment:
The `importing` doc needs to be updated to include publishing and importing Collections. Maybe just mention that it's different, and link to the appropriate spot in `creating_collections.rst`